api/files/serializers.py

```diff
@@ -88,6 +111,7 @@ class FileSerializer(JSONAPISerializer):
         'size',
         'provider',
         'last_touched',
+        'tags',
```

Comment:
Or I thought so. I know we had chatted about it, but I can't find the discussion or the point at which tests failed and he changed the code back. Hmm.